docs/build/inferencing.md

```diff
@@ -480,7 +480,7 @@ pip3 install numpy
 # Build the latest cmake
 mkdir /code
 cd /code
-wget https://cmake.org/files/v3.13/cmake-3.16.1.tar.gz;
+wget https://cmake.org/files/v3.16/cmake-3.16.1.tar.gz;
```

Comment:
The URL on the left is invalid. 

This doc should be updated to use cmake from pypi to avoid such issues.

Reply:
So, the whole cmake section should be replaced with a call to pip install cmake?

https://onnxruntime.ai/docs/build/inferencing.html#native-compiling-on-linux-arm-device

Reply:
Yes.

Reply:
I was wrong. cmake on pypi doesn't have an ARM32 package:

https://pypi.org/project/cmake/#files

So it has to be compiled from source.

Reply:
Is the current text correct now?